embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -5,6 +5,12 @@ overwrite_optimize=""
 
 while true; do
     case "$1" in
+        -E*)
+            external_script="${1#-E}"
+            echo "Running an external shell script: $external_script"
+            source $external_script
```

Comment:
nit  `/bin/dash` doesn't work `source`. I think it is better to use `.` instead of `source`.

```
$ source /tmp/test.sh
dash: 1: source: not found
```

```
$ . /tmp/test.sh
$
```

Reply:
I thought `source` and `.` have no differences... Can you confirm the `.` usage of yours change your environment variables on the parent shell, or not? I guess that is not changing.